superchain/configs/sepolia-dev-0/superchain.yaml

```
@@ -0,0 +1,8 @@
+name: Sepolia Dev 0
+l1:
+  chain_id: 11155111
+  public_rpc: https://ethereum-sepolia-rpc.allthatnode.com
+  explorer: https://sepolia.etherscan.io
+
+protocol_versions_addr: null # todo
```

we just deployed them
```suggestion
protocol_versions_addr: "0x252CbE9517F731C618961D890D534183822dcC8d"
```